communities/plymouth-university/community-assessment.md

```diff
@@ -0,0 +1,122 @@
+# Plymouth University
+
+Plymouth University has been having troubles with the tech community for a number of years. We've seen a seady decline in the number of tech based societies and events since around 2012. We are now in a situation where the last community left is the Game Dev Society (GDSoc). Thankfully there are still a few pockets of students who are enthusiastic about various strands of tech.
```

Comment:
That would be fantastic, you have my email if you'd like to connect us that way?